python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -893,7 +930,7 @@ def run(
                 illumination_correction,
             )
 
-            result.psf_stars_footprints, result.background, _ = self._compute_psf(
+            result.psf_stars_footprints, result.background, _, adaptiveDetResStruct = self._compute_psf(
```

Comment:
This task is pretty consistently snake_case; it'd be good to continue that here.